### PR TITLE
 update jetty client and server example to use updated api.

### PR DESCRIPTION
**Pull request by rghetia**
- use client api that supports ssl context.
- use default value for publicEndpoint on server.
- Add comment on how to use B3 format on both sides, client and server.

### CODE REVIEW

File: examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java [hunk 1]
```diff
@@ -179,18 +183,35 @@ private static void initTracing() {
...
+    contextHandler.addServlet(sh, "/request/*");
+
+    // Enable tracing by adding OcHttpServleFilter for all path
+    contextHandler.addFilter(OcHttpServletFilter.class, "/*", EnumSet.of(DispatcherType.REQUEST));
+
+    // Uncomment following line to use B3Format for trace context propagation.
+    // contextHandler.setAttribute(
+    //    OC_TRACE_PROPAGATOR, Tracing.getPropagationComponent().getB3Format());
+
+    // By default publicEndpoint parameter is set to false and incoming trace context is added as
```
Comment (dinooliva) [inline examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java]: I may be misreading this but it sounds like the incoming trace context is added as a parent link in both the true and false cases.

Comment (rghetia) [reply examples/src/main/java/io/opencensus/examples/http/jetty/server/HelloWorldServer.java]: you are right. the first one should be parent.

File: examples/src/main/java/io/opencensus/examples/http/jetty/client/HelloWorldClient.java [hunk 2]
```diff
@@ -68,17 +71,37 @@ public static void main(String[] args) throws Exception {
     initTracing();
     initStatsExporter();
 
-    OcJettyHttpClient httpClient = new OcJettyHttpClient();
+    // Create http client that will trace requests. By default trace context is propagated using
+    // using w3c TraceContext propagator.
```
Comment (songy23) [inline examples/src/main/java/io/opencensus/examples/http/jetty/client/HelloWorldClient.java]: Remove the duplicated "using"

Comment (rghetia) [reply examples/src/main/java/io/opencensus/examples/http/jetty/client/HelloWorldClient.java]: done.

**Pull request merged**